scripts/conversion_script.py

```diff
@@ -36,6 +36,46 @@ def __init__(self, path="", host="", keep=False):
         self.path = path
         self.host = host
         self.keep = keep
+        self.backup_suffix = ".backup"
+
+    def create(self, data):
+        response = urlopen(self.host)
+        data = response.read()
+        directory = os.path.dirname(self.path)
+        if not os.path.exists(directory):
+            print("Creating directory at '%s'" % directory)
+            os.makedirs(directory, mode=0o755)
+
+        print("Writing file to destination: '%s'" % self.path)
+        with open(self.path, mode="w") as handler:
+            handler.write(data)
+            os.chmod(self.path, 0o644)
```

Comment:
if the method gets data from optional param it won't call urlopen, now for our use cases this param is `None` in default, because all files are taken from URL